### PR TITLE
Introduction to Cloud Management

### LINKED ISSUE #2932: Cloud Management in Cloud Computing
### Introduction
Cloud management is the medium that allows administration control of technologies and software. Cloud management cares for products and services. 
Cloud management is centralized. It manages resources, assets, data storage, applications, etc.

### Key Takeaways
What cloud management is.
Cloud management designs (platform).
Cloud management challenges.
Functionalities a good cloud manager should have.

### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. *Use keyword research to gain your article higher ranking.*

### Introduction paragraph (2-3 paragraphs):
Please write the Introductory paragraph(s), that would be included in your article. We will use this writing snippet to help us assess overall quality before approval. We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.
Take your time and write the content as you would intend to get it published. 

### Key takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article? Use this area to get your ideas down on the bulk of your article or tutorial.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by kwennB**
Cloud management caters to products and services. It allows the operations of the kinds of cloud infrastructure. Also, it manages products and services, including its flexibility. The scalability, security, and adaptation when the wants arise.

### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [ ] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [ ] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [ ] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [ ] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [ ] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [ ] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [ ] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [ ] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [ ] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [ ] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [ ] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [ ] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [ ] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

This fixes #2932

### CODE REVIEW

**Comment by OMONDIELLY** [comment]
I'll like to review this article @hectorkambow.

File: content/articles/cloud-management/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+layout: engineering-education
+title: Cloud Management
+description: Cloud management is the control of products and services running in the cloud. It caters for these products and services. In this article we will get to understand how to choose the best cloud platforms. Also, we'll discuss challenges of cloud management and ways to beat them.
+author: anyebe-blessing-ene
+date: 2021-08-06T00:00:00-10:30
+topics: {Cloud}
+excerpt_seperator: <!--more-->
+images: 
+---
+In this article, we will discuss how cloud management and understand what there is to know about it.
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:11]: You could start with a more descriptive introduction.

Why does the reader want to know about cloud management? What is it? How will it help the reader?  Then, share the goals/objectives of this tutorial

You must make the reader feel interested about this article.

File: content/articles/cloud-management/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+<!--more-->
+
+### Table of Contents
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:14]:
```suggestion
### Table of contents
```

File: content/articles/cloud-management/index.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- Cloud Management
+- Cloud Management Platforms
+- Cloud Management Challenges
+- Cloud Management Benefits
+- Functionalites of a decent cloud manager
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:19]: Add clickable links. Refer to my articles to understand them better.

File: content/articles/cloud-management/index.md (hunk 1 above)
Comment (srishilesh) [reply content/articles/cloud-management/index.md:11]: Start with cloud. Explain how apps migrated from local to cloud.

File: content/articles/cloud-management/index.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+### Prerequisities 
+Basic knowledge of cloud computing
+
+Let's dive right in by learning about the concept
+
+### Cloud Management
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:26]: Before you get into this topic, state about the current scenario.

Explain the current drawback. What's the need to manage? 

Don't just state facts. Walk the reader through the problem, and give a solution

File: content/articles/cloud-management/index.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+- CA technologies
+- Microfocus
+- ServiceNow
+
+### Cloud Management Challenges
+
+In this century, enterprises have migrated from storing data or their applications online. Neither are uploads stored through storage device and servers. Given that, using cloud storage seems more efficient. Regardless cloud management doesn’t come without its setbacks and challenges. One major oblivious factor is the resources it costs but that's one of the few. Thus, we'd like to grasp about its challenges. Also, the simplest solutions required for them. Hence we are able to completely maximize the advantages of cloud management. 
+Here some of the challenges of cloud management and ways to beat them; 
+
+**Cost Management**
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:72]: Convert them into sub-topics with headers `####`

Comment (srishilesh) [reply content/articles/cloud-management/index.md:72]: Do the same for below sub-topics too

File: content/articles/cloud-management/index.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+ While choosing a cloud provider, understand how the customer can reach the service. Most of them use an embedded chatbot, which can not be effective for support. It should be less strenuous to induce in-tuned with the web of the cloud provider you decide on.
+
+**How Easy it's to Manage**
+ Many cloud services need the employment of third-party apps. When choosing, you ought to consider how easy is it to integrate between each of the services. Consider the time it'll take your enterprise to prepare the mixing. * They must be cloud services that incorporate self-service features.
+
+
+### Conclusion
+Cloud management is an essential knowledge. We need it to grasps the workings of cloud computing while starting out. It does not matter if you have an enterprise or intend on getting one. If you are tech inclined and used mobile devices. You should get acquainted to the modern day storage systems.  
+
+Thank you for reading!
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:137]: Add a reference links for further reading

**Review by srishilesh — CHANGES_REQUESTED** [review]
@kwennB 

Nice article.

I've suggested a few changes on formatting and enhancements. Please ping me when done.

Thanks.

**Comment by kwennB** [comment]
Thank you for the feedback @srishilesh 
I will see to requested changes.

**Comment by hectorkambow** [comment]
@kwennB 
Upon a quick glance this article seem rather light on substance. 

cc @srishilesh

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by srishilesh** [comment]
@kwennB 

Any updates on this PR?

**Comment by kwennB** [comment]
@srishilesh

I am working on it. 
I'll push soonest.
I sent you a message on slack.

**Comment by srishilesh** [comment]
@kwennB 

Are all the changes done? Can I start my review?

**Comment by kwennB** [comment]
@srishilesh 

Yes. You can start the review.

File: content/articles/cloud-management/index.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+- Event-driven architecture
+- Cloud providers must have the power to integrate a client’s cloud computing needs. With their on-premise IT operations in an seamless fashion.
+
+### Better Ways to Manage Cloud Resources
+* Monitor your cloud space, the essential purpose of cloud management
+* Use the best security
+* Create the infrastructure that will suit your business 
+* Modify storage environment
+* Use automation 
+* Keep records: Write down what you do, document your experience.
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:125]: `*` -> `-`

File: content/articles/cloud-management/index.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+#### How Easy it's to Manage**
+ Many cloud services need the employment of third-party apps. When choosing, you ought to consider how easy is it to integrate between each of the services. Consider the time it'll take your enterprise to prepare the mixing. * They must be cloud services that incorporate self-service features.
+
+ ### References
+1. [Cloud Migration](https://searchcloudcomputing.techtarget.com/definition/cloud-migration)
+2. [Importance of Cloud Computing](https://www.hashmicro.com/blog/reasons-why-cloud-computing-is-important-to-business/#:~:text=Cloud%20Computing%20model%20enables%20your,files%20at%20the%20same%20time)
+3. [Best Practice for using Cloud](https://www.software.ac.uk/best-practice-using-cloud-research)(sustainability institue, 2020)
+
+### Conclusion
+Cloud management is an essential knowledge. We need it to grasps the workings of cloud computing while starting out. It does not matter if you have an enterprise or intend on getting one. If you are tech inclined and used mobile devices. You should get acquainted to the modern day storage systems.
```
Comment (srishilesh) [inline content/articles/cloud-management/index.md:168]: Add key takeaways

**Review by srishilesh — CHANGES_REQUESTED** [review]
@kwennB 

After going through your changes, I noticed that you have not gone through all my suggestions. There are a few more changes left.

Please ping me when everything is done.

Thanks.

**Comment by kwennB** [comment]
@srishilesh 

Noted 

Will do.

**Comment by kwennB** [comment]
@srishilesh 

I have made the suggested changes. Kindly check them.

Thank you.

**Comment by hectorkambow** [comment]
@srishilesh  

Just looking for a update on this article 👍

**Comment by srishilesh** [comment]
@hectorkambow 

I'll complete the review by the end of the day.

**Review by srishilesh — APPROVED** [review]
@kwennB 

I've made several changes to formatting and flow. Make sure to go through all my commits.

Moving for the final review.

Thanks.

**Comment by hectorkambow** [comment]
@kwennB 

Thank you for submitting your topic to the EngEd program and jumping on your revisions. 
As a rule of thumb - we can only accept articles (pull requests) that are fully polished and complete articles - articles that are ready to be published.
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 

Please be sure to go over any edits made by your peer reviewer to improve future articles and reviews and prevent any PR closure.

**Comment by hectorkambow** [comment]
Overall nice job - made some minor edits to the format and for the flow. 
Easy to follow along - please go over any edits made to improve future articles and reviews. 

👍

**Comment by kwennB** [comment]
@hectorkambow 
@srishilesh 

Thank you. All the feedback have been noted.

**Pull request merged**